src/api/public/apidocs-new/paths/missing_owner.yaml

```
@@ -0,0 +1,62 @@
+get:
+  summary: |
+    Search for packages that miss the definition for a by default
+    responsible person or group.
+  description: |
+    Search for packages that miss the definition for a by default
+    responsible person or group.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: project
+      schema:
+        type: string
+      description: Only return results for a given project.
+    - in: query
+      name: filter
+      schema:
+        type: string
+      description: |
+        A comma seperated list of role names that should be
```

```suggestion
        A comma separated list of role names that should be
```
Typo in `separated`.